### DISCUSSION TITLE
Video Effects for Source Mirrors?

### DISCUSSION THREAD

**Discussion by MetacLeod** [Xaymar/obs-stream-effects#831] [Q&A] [answered]
Is it possible to use a non-RTX background remove plugin with a source mirror? I don't have an rtx card and have the background remove plugin, but because neither nested scenes nor scene mirrors have the ability to use video effects (not effect filters), I can't use the background remove plugin. If this were possible I would be able to do so many things! Specifically I'd like to have two of the same camera on top of each other with separate plugins/effects on them to [achieve this.](https://youtu.be/3UNhCrFXGZw?t=512)

I hope this is possible! And if not I hope maybe someday it will be! Thanks for all your hard work, we all appreciate it so much!
Cheers

**Comment by Xaymar** [comment, marked as answer]
Unfortunately it is not possible to apply _CPU_ filters to things entirely residing on the _GPU_. You'll have to ask the author of the other plugin to make their filter compatible with the GPU filtering path.

**Reply by MetacLeod** [reply]
Thank you so much!
